customization-base/src/main/java/com/azure/autorest/customization/ConstructorCustomization.java

```diff
@@ -134,26 +136,35 @@ public ConstructorCustomization replaceBody(String newBody) {
 
     /**
      * Replaces the body of the constructor and adds any additional imports required by the new body.
+     * <p>
+     * If the new body contains {@code this()} or {@code super()} constructor calls, use
+     * {@link #replaceBody(NodeList, List)} instead.
      *
      * @param newBody New constructor body.
      * @param importsToAdd Any additional imports required by the constructor. These will be custom types or types that
      * are ambiguous on which to use such as {@code List} or the utility class {@code Arrays}.
      * @return A new ConstructorCustomization representing the updated constructor.
      */
     public ConstructorCustomization replaceBody(String newBody, List<String> importsToAdd) {
```

Comment:
I think previously language server would be able to automatically add some missing imports used in the new body.
I guess java-parser won't.